README.md

```diff
@@ -62,6 +64,28 @@ docker create \
     cpoppema/docker-flexget
 ```
 
+### docker-compose
+
+```
+---
+  flexget:
```

Comment:
If you can turn this into a fully working yml with the following bit that'd be great :smile:

```
version: "2"

services:
  flexget:
    .....
```

Reply:
done :)